test/base/driver_util.py

```diff
@@ -442,7 +442,7 @@ def setup_shed_tools_for_test(app, tmpdir, testing_migrated_tools, testing_insta
             tool_configs.remove(relative_migrated_tool_panel_config)
         for installed_tool_panel_config in INSTALLED_TOOL_PANEL_CONFIGS:
             tool_configs.append(installed_tool_panel_config)
-        from galaxy import tools  # noqa, delay import because this brings in so many modules for small tests
+        from galaxy import tools  # delay import because this brings in so many modules for small tests # noqa
```

Comment:
Cool, was doing the same! My ack found:
lib/galaxy/model/mapping.py
lib/galaxy/webapps/galaxy/controllers/dataset.py
lib/galaxy/webapps/galaxy/controllers/forms.py
lib/galaxy/webapps/galaxy/controllers/library_common.py
scripts/db_shell.py